addons/hr/models/hr_employee_base.py

```diff
@@ -40,6 +40,11 @@ class HrEmployeeBase(models.AbstractModel):
     email = fields.Char(related="user_id.email")
     work_contact_id = fields.Many2one('res.partner', 'Work Contact', copy=False)
     work_location_id = fields.Many2one('hr.work.location', 'Work Location', domain="[('address_id', '=', address_id)]")
+    name_work_location_display = fields.Char("Displayed Work Location", compute="_compute_type_name_work_location_display")
```

Comment:
Strange naming for this field.
Isn't it just `work_location_name`?

Reply:
I forget where I picked the strange name, I guess at first it was intended to be diff from `work_location.name` since it is computed and it is a display name rather than `work_location.name`. But I agree this is confusing. I would rename it and if it bothers someone we can change it back later.